system-tests.js

```diff
@@ -238,4 +242,28 @@ run('ern platform list')
 run('ern platform plugins list')
 run('ern platform plugins search react-native')
 
-afterAll()
+afterAll() */
+
+// System Tests for API
+console.log(`#### Path to Default schema ${pathToDefaultSchema}`)
+console.log(`#### Path to Default API ${pathToDefaultAPI}`)
+console.log(`#### Path to generated API ${workingDirectoryPath}`)
+run(`ern create-api ${apiName} -p ${apiPkgName} --skipNpmCheck --schemaPath ${pathToDefaultSchema}`)
+assert(areSameDirectoriesContent(pathToDefaultAPI, workingDirectoryPath), 'Generated API differ from reference fixture !')
+console.log('Done, test passes API !!!!!!!!!')
+
+// Tests for API IMPL Native
+console.log(`### Path to Native-impl ${pathToApiImplNative}`)
+console.log(`### Path to workingDirectoryPath ${workingDirectoryPath}`)
+process.chdir(workingDirectoryPath)
+run(`ern create-api-impl ${movieApi} -p ${movieApiImplPkgName} --skipNpmCheck --nativeOnly --outputDirectory ${workingDirectoryPath} --force`)
```

Comment:
can we please adapt the existing api-impl test to work with folder content comparison? To limit the test run time